### ISSUE TITLE
Custom repeater not working in columns (Elementor 3.0.x)

### ISSUE THREAD

**Issue by GregorCodex** [elementor/elementor#12270]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description

Just like in this ticket https://github.com/elementor/elementor/issues/12221 the same happens with columns. 

After adding repeater control to column settings in Elementor 3.0 we get js errors when trying to remove/delete widgets from the column (the same happens with section as well).

## Steps to reproduce

we have made a fresh new install with a simple plugin containing following code:

```
use Elementor\Controls_Manager;
add_action( ‘elementor/element/column/section_style/before_section_end’, ‘my_test_repeater’, 10, 2 );
function my_test_repeater( $element, $args ) {
$element->add_control(
‘my_test_title’,
[
‘label’ => ‘Test Title’,
‘type’ => Controls_Manager::HEADING,
]
);

$element->add_control(
‘my_width’,
[
‘label’ => ‘Width’,
‘type’ => Controls_Manager::NUMBER,
]
);

$repeater = new \Elementor\Repeater();

$repeater->add_control(
‘width’,
[
‘label’ => ‘Width’,
‘type’ => Controls_Manager::NUMBER,
]
);

$element->add_control(
‘my_test_repeater’,
[
‘type’ => Controls_Manager::REPEATER,
‘fields’ => $repeater->get_controls(),
‘title_field’ => ‘Test Repeater’
]
);
}
```
This plugin adds repeater control to column settings.

After adding a new section/column with some widgets to the page and by trying to remove / delete this widgets we get following js errors in console:

http://prntscr.com/u66ypd

common.min.js?ver=3.0.1:2 TypeError: Cannot set property children of #<Container> which has only a getter

And widgets cannot be removed – nothing happens, widgets remain in column.

**Comment by dmeijide** [comment]
Also, the History and Revisions does not work properly after the update. It does not remove the items you created when you go back.

**Comment by quentinaurat** [comment]
As you can see, I get exactly the same issue with the new buid 3.0.2 : [https://prntscr.com/u6i2vd](url)
This is pretty weird because it happens usually on the first section of the template if you have multiple sections. I've made the test with a local existing test site with content upgraded from Elementor 2.X.
This is a really big issue because you "virtually" loose your content, and you canot recreate the section or some kind of refresh...

**Comment by VadymPogorelov** [comment]
It seems that the problem is in the number of fields of the REPEATER type:
- the 'parallax_blocks' repeater is added to all sections and columns https://www.screencast.com/t/neFH1v8yI
- the 'audio_effects' repeater is also added to all sections, columns, and other shortcodes https://www.screencast.com/t/bBbhnQUqEjP7
In this combination, everything works without errors.
If I comment on ANY of these fields, the error appears: https://www.screencast.com/t/XoG8WNvcNY8
Comment on BOTH fields-disappears!!!

If one field is added, deleting rows is also quite funny: https://www.screencast.com/t/iolzJVTI
And it is accompanied by the same error

P.S. After the 3.0 update, the editor started loading and initializing much slower!

**Comment by 319heads** [comment]
Same issues here.

**Comment by sermalefico** [comment]
@shilo-ey  Same issues in 3.0.3 update

**Comment by VadymPogorelov** [comment]
The problem with REPEATER fields remained after update 3.0.3 - see my comment above: https://github.com/elementor/elementor/issues/12270#issuecomment-681822281

Two REPEATER fields have been added to the columns:
https://www.screencast.com/t/ffaJD3wu
https://www.screencast.com/t/NC2dmKSriO

In this case there are no errors: https://www.screencast.com/t/cp5bUCTCb

If you remove (comment) any of the fields (i.e. leave only one):
https://www.screencast.com/t/z7xz315x
https://www.screencast.com/t/y4JM1ydiXQ3
errors appear and deleting blocks stops working. Also, when deleting sections, the internal blocks are deleted first (one at a time) and only then the entire section: https://www.screencast.com/t/g36XccpQ7cc

If you remove (comment) both fields (i.e. do not add any REPEATER type fields at all), everything works well again

**Comment by shilo-ey** [comment]
I merged this issue with #12242, which is on the same subject so we could deal with it more efficiently. 
Please feel free to continue the conversation there.

Thanks for your feedback!
